### PR TITLE
chore: prepare Tokio v1.41.0

### PR DESCRIPTION
**Pull request by mox692**
# 1.41.0 (Oct 22th, 2024)

### Added

- metrics: stabilize `global_queue_depth` ([#6854], [#6918])
- net: add conversions for unix `SocketAddr` ([#6868])
- sync: add `watch::Sender::sender_count` ([#6836])
- sync: add `mpsc::Receiver::blocking_recv_many` ([#6867])
- task: stabilize `Id` apis ([#6793], [#6891])

### Added (unstable)

- metrics: add H2 Histogram option to improve histogram granularity ([#6897])
- metrics: rename some histogram apis ([#6924])
- runtime: add `LocalRuntime` ([#6808])

### Changed

- runtime: box futures larger than 16k on release mode ([#6826])
- sync: add `#[must_use]` to `Notified` ([#6828])
- sync: make `watch` cooperative ([#6846])
- sync: make `broadcast::Receiver` cooperative ([#6870])
- task: add task size to tracing instrumentation ([#6881])
- wasm: enable `cfg_fs` for `wasi` target ([#6822])

### Fixed

- net: fix regression of abstract socket path in unix socket ([#6838])

### Documented

- io: recommend `OwnedFd` with `AsyncFd` ([#6821])
- io: document cancel safety of `AsyncFd` methods ([#6890])
- macros: render more comprehensible documentation for `join` and `try_join` ([#6814], [#6841])
- net: fix swapped examples for `TcpSocket::set_nodelay` and `TcpSocket::nodelay` ([#6840])
- sync: document runtime compatibility ([#6833])

[#6793]: https://github.com/tokio-rs/tokio/pull/6793
[#6808]: https://github.com/tokio-rs/tokio/pull/6808
[#6810]: https://github.com/tokio-rs/tokio/pull/6810
[#6814]: https://github.com/tokio-rs/tokio/pull/6814
[#6821]: https://github.com/tokio-rs/tokio/pull/6821
[#6822]: https://github.com/tokio-rs/tokio/pull/6822
[#6826]: https://github.com/tokio-rs/tokio/pull/6826
[#6828]: https://github.com/tokio-rs/tokio/pull/6828
[#6833]: https://github.com/tokio-rs/tokio/pull/6833
[#6836]: https://github.com/tokio-rs/tokio/pull/6836
[#6838]: https://github.com/tokio-rs/tokio/pull/6838
[#6840]: https://github.com/tokio-rs/tokio/pull/6840
[#6841]: https://github.com/tokio-rs/tokio/pull/6841
[#6846]: https://github.com/tokio-rs/tokio/pull/6846
[#6854]: https://github.com/tokio-rs/tokio/pull/6854
[#6867]: https://github.com/tokio-rs/tokio/pull/6867
[#6868]: https://github.com/tokio-rs/tokio/pull/6868
[#6870]: https://github.com/tokio-rs/tokio/pull/6870
[#6881]: https://github.com/tokio-rs/tokio/pull/6881
[#6890]: https://github.com/tokio-rs/tokio/pull/6890
[#6891]: https://github.com/tokio-rs/tokio/pull/6891
[#6897]: https://github.com/tokio-rs/tokio/pull/6897
[#6918]: https://github.com/tokio-rs/tokio/pull/6918
[#6924]: https://github.com/tokio-rs/tokio/pull/6924

### CODE REVIEW

**Comment by mox692** [comment]
As mentioned in [#6915](https://github.com/tokio-rs/tokio/issues/6915), I guess there could be a chance to change the metrics name of `injection_queue_depth` if we prefer.

File: tokio/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,61 @@
...
+
+- metrics: stabilize `global_queue_depth` ([#6854], [#6918])
+- net: add conversions for unix `SocketAddr` ([#6868])
+- sync: add `watch::Sender::sender_count` ([#6836])
+- sync: add `mpsc::Receiver::blocking_recv_many` ([#6867])
+- task: stabilize `Id` apis ([#6793], [#6891])
+
+### Added (unstable)
+
+- runtime: add `LocalRuntime` ([#6808])
```
Comment (hds) [inline tokio/CHANGELOG.md:13]: Should we mention that this is unstable at the moment?

Comment (Darksonn) [reply tokio/CHANGELOG.md:13]: It seems like it's already under an "unstable" section?

Comment (hds) [reply tokio/CHANGELOG.md:13]: Oh, you're right, I completely missed that! Thanks! 🤦‍♀️

File: tokio/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,61 @@
...
+
```
Comment (hds) [inline tokio/CHANGELOG.md:14]: This PR got merged today:

```suggestion
- metrics: add H2 Histogram option to improve histogram granularity ([#6897])

```

File: tokio/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,61 @@
...
+[#6846]: https://github.com/tokio-rs/tokio/pull/6846
+[#6854]: https://github.com/tokio-rs/tokio/pull/6854
+[#6867]: https://github.com/tokio-rs/tokio/pull/6867
+[#6868]: https://github.com/tokio-rs/tokio/pull/6868
+[#6870]: https://github.com/tokio-rs/tokio/pull/6870
+[#6881]: https://github.com/tokio-rs/tokio/pull/6881
+[#6890]: https://github.com/tokio-rs/tokio/pull/6890
+[#6891]: https://github.com/tokio-rs/tokio/pull/6891
+[#6918]: https://github.com/tokio-rs/tokio/pull/6918
+
```
Comment (hds) [inline tokio/CHANGELOG.md:58]: This PR got merged today:

```suggestion
[#6897]: https://github.com/tokio-rs/tokio/pull/6897

```

**Review by hds — COMMENTED** [review]
Add items for https://github.com/tokio-rs/tokio/pull/6897 to the release.

File: tokio/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,63 @@
...
+- sync: add `watch::Sender::sender_count` ([#6836])
+- sync: add `mpsc::Receiver::blocking_recv_many` ([#6867])
+- task: stabilize `Id` apis ([#6793], [#6891])
+
+### Added (unstable)
+
+- metrics: add H2 Histogram option to improve histogram granularity ([#6897])
+- runtime: add `LocalRuntime` ([#6808])
+
+### Changed
```
Comment (Darksonn) [inline tokio/CHANGELOG.md:16]: We merged https://github.com/tokio-rs/tokio/pull/6924 too.

Sorry for so many edits :) let's not make any more.

File: tokio/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,63 @@
+# 1.41.0 (Oct 19th, 2024)
```
Comment (Darksonn) [inline tokio/CHANGELOG.md:1]:
```suggestion
# 1.41.0 (Oct 22nd, 2024)
```

**Review by Darksonn — APPROVED** [review]
Thank you.

**Pull request merged**